apps/frontend/src/components/MessageCard.vue

```diff
@@ -38,7 +38,7 @@
                     <main class="max-w-[500px] break-all flex justify-between min-h-[36px]">
                         <MessageContent
                             :message="message"
-                            :key="String(message.id)"
+                            :key="String(message.id) + message.body"
```

Comment:
not needed as key will be too long

Reply:
also id is unique

Reply:
this is needed 
Because the type stayed the same the message didn't get reloaded by vue. So I changed the key so that when the body changes the key also gets updated and vue gets reloaded (best practice for re-render of Vue is with :key according to the docs)

Reply:
yes key is needed but message.id should be enough

Reply:
if i remove the message.body it doesn't work